roles/edpm_neutron_sriov/tasks/install.yml

```diff
@@ -15,13 +15,16 @@
 # under the License.
 
 - name: Create neutron-sriov-agent directories
+  become: true
   ansible.builtin.file:
     path: "{{ item.path }}"
     setype: "container_file_t"
     state: directory
+    owner: "{{ item.owner | default(ansible_user) }}"
+    group: "{{ item.group | default(ansible_user) }}"
     mode: "{{ item.mode | default(omit) }}"
   loop:
-    - {'path': "/var/lib/openstack/config/containers", "mode": "0750"}
+    - {'path': "/var/lib/openstack/config/containers", "mode": "0755"}
     - {'path': "/var/lib/neutron", "mode": "0750"}
```

Comment:
thise seems to work but you might want to chge the owner/group default to default(omit) and pass ansible_user for the files that should that user otherwise for /var/lib/neutron it probably should be owned by neutron

it will get changed when the neutron contianers start but you don't want to be changing it back to anisble_user while the contiaenrs are running

Reply:
this might be ok as is im just highlight this line since the user/group will now change and I'm not sure that was intentional

Reply:
> thise seems to work but you might want to chge the owner/group default to default(omit) and pass ansible_user for the files that should that user otherwise for /var/lib/neutron it probably should be owned by neutron
> 
> it will get changed when the neutron contianers start but you don't want to be changing it back to anisble_user while the contiaenrs are running

done

Reply:
> this might be ok as is im just highlight this line since the user/group will now change and I'm not sure that was intentional

We have discussed like, this directories should be owned by 'cloud-admin'. so used ansible_user.

Reply:
@SeanMooney 
For workaround, can we merge this PR, will work and create another PR for above changes later.
and also need to remove become in playbook level.

Reply:
> thise seems to work but you might want to chge the owner/group default to default(omit) and pass ansible_user for the files that should that user otherwise for /var/lib/neutron it probably should be owned by neutron
> 
> it will get changed when the neutron contianers start but you don't want to be changing it back to anisble_user while the contiaenrs are running

Tested and working fine when i use default(omit) instead of ansible_user but wner and group will be root for dir '/var/lib/openstack/config'. will it be fine?

Reply:
no /var/lib/openstack/config must be ownewd by ansible_user

the fact it is owned by root is what you need to fix in this series

Reply:
> no /var/lib/openstack/config must be ownewd by ansible_user
> 
> the fact it is owned by root is what you need to fix in this series

Ack

Reply:
you need to chagne it to 

- {'path': "/var/lib/openstack/config/containers", "mode": "0755", user: "{{ ansible_user }}", group: "{{ ansible_user }}" }

Reply:
same for 
edpm_neutron_sriov_agent_config_dir and /var/log/containers/stdouts

Reply:
done